src/corppa/utils/path_utils.py

```diff
@@ -118,16 +118,18 @@ def get_image_relpath(work_id, page_num):
         raise ValueError(f"Unsupported source '{source}'")
 
 
-def page_number(filename: pathlib.Path) -> str:
-    """Extract and return the page number from a :class:`pathlib.Path`
-    file for content from a single page (e.g., image or text). Returns the
-    page number as a string with leading zeros. (Note: logic is curently
+def get_page_number(filename: str) -> str:
```

Comment:
Was there a reason for switching from `pathlib.Path` to `str` as input? The equivalent of `os.path.splitext` is to just use `Path.stem` & `Path.ext`.

Reply:
I misunderstood a previous comment, I thought you requested that I make that change. 

Just switched it back to pathlib. Shall we get this merged?

Reply:
Looks good to me! Sorry for the confusion.